Lib/test/test_asyncio/test_base_events.py

```diff
@@ -91,6 +91,9 @@ def test_ipaddr_info(self):
         self.assertIsNone(
             base_events._ipaddr_info('1.2.3.4', 1, UNSPEC, 0, 0))
 
+        if not support.IPV6_ENABLED:
+            return
```

Comment:
I don't like this approach, because it's hard to see that the rest of the test is skipped. Please either split the test function into two or move the rest of the code into the ``if not supported.IPV6_ENABLED`` block.

Reply:
Okay. I am willing to make such changes, but I would like to hear what @asvetlov thinks.

Reply:
@ZackerySpytz please make a PR with changes suggested by @tiran 

My first reaction was opening the source code to make sure that the `return` doesn't make harm.
On the other hand, I've committed the PR as is because you proposed a minimal possible diff.

Reply:
Okay. I have created GH-14535.